charts/collectorset-controller/templates/collector-rbac.yaml

```diff
@@ -115,6 +115,12 @@ rules:
     verbs:
       - list
       - get
+  - apiGroups:
+      - "apiextensions.k8s.io"
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - "*"
```

Comment:
good to mention list of verbs than wildcard - it opens gates to access everything - write access too which is not recommended.